src/controller.js

```diff
@@ -47,6 +50,28 @@ class ChatbotCtrl {
 
     this.$scope.$on('ovh-chatbot:open', () => this.open());
     this.$scope.$on('ovh-chatbot:opened', () => this.focusInput());
+
+    const config = this.config || {};
+    if (isString(this.url)) {
+      config.url = this.url;
+    }
+    this.ChatbotService.setConfig(config);
+  }
+
+  isInvisibleMessage(text) {
+    return this.qualifyMessage(text) === this.MESSAGE_QUALITY.invisible;
+  }
+
+  qualifyMessage(text) {
+    if (text.startsWith('##')) {
```

Comment:
Don't you use a 2API service ? Because this should be done in the 2API then :)

Reply:
Well I also need to pass that filter on user's messages, so this need to stay in the front :/

Reply:
It is not possible to have a `type` property (or something else) in the payload ?

Reply:
That's already what I'm doing. I'm analyzing every message (received from the server, or typed by the user) and set a "quality" property to each of'em.